src/cascadia/TerminalApp/ActionArgs.cpp

```diff
@@ -15,3 +15,186 @@
+        }
+        else
+        {
+            return winrt::hstring{ fmt::format(L"{}, {}", RS_(L"NewTabCommandKey"), newTerminalArgsStr) };
+        }
+        return L"NewTabArgs";
+    }
+
+    winrt::hstring SwitchToTabArgs::GenerateName() const
+    {
+        return winrt::hstring{ fmt::format(L"{}, index:{}", RS_(L"SwitchToTabCommandKey"), _TabIndex) };
+    }
+
+    winrt::hstring ResizePaneArgs::GenerateName() const
+    {
+        winrt::hstring directionString;
+        switch (_Direction)
+        {
+        case Direction::Left:
+            directionString = RS_(L"DirectionLeft");
+            break;
+        case Direction::Right:
+            directionString = RS_(L"DirectionRight");
+            break;
+        case Direction::Up:
+            directionString = RS_(L"DirectionUp");
+            break;
+        case Direction::Down:
+            directionString = RS_(L"DirectionDown");
+            break;
+        }
+        return winrt::hstring{ fmt::format(RS_(L"ResizePaneWithArgCommandKey").c_str(), directionString) };
+    }
+
+    winrt::hstring MoveFocusArgs::GenerateName() const
+    {
+        winrt::hstring directionString;
+        switch (_Direction)
+        {
+        case Direction::Left:
+            directionString = RS_(L"DirectionLeft");
+            break;
+        case Direction::Right:
+            directionString = RS_(L"DirectionRight");
+            break;
+        case Direction::Up:
+            directionString = RS_(L"DirectionUp");
+            break;
+        case Direction::Down:
+            directionString = RS_(L"DirectionDown");
+            break;
+        }
+        return winrt::hstring{ fmt::format(RS_(L"MoveFocusWithArgCommandKey").c_str(), directionString) };
+    }
+
+    winrt::hstring AdjustFontSizeArgs::GenerateName() const
+    {
+        if (_Delta < 0)
+        {
+            return _Delta == -1 ? winrt::hstring{ fmt::format(RS_(L"DecreaseFontSizeCommandKey").c_str()) } :
+                                  winrt::hstring{ fmt::format(RS_(L"DecreaseFontSizeWithAmountCommandKey").c_str(), -_Delta) };
+        }
+        else
+        {
+            return _Delta == 1 ? winrt::hstring{ fmt::format(RS_(L"IncreaseFontSizeCommandKey").c_str()) } :
+                                 winrt::hstring{ fmt::format(RS_(L"IncreaseFontSizeWithAmountCommandKey").c_str(), _Delta) };
+        }
+    }
+
+    winrt::hstring SplitPaneArgs::GenerateName() const
+    {
+        std::wstringstream ss;
+        if (_SplitMode == SplitType::Duplicate)
+        {
+            ss << RS_(L"DuplicatePaneCommandKey").c_str();
+        }
+        else
+        {
+            ss << RS_(L"SplitPaneCommandKey").c_str();
+        }
+        ss << L", ";
+
+        switch (_SplitStyle)
+        {
+        case SplitState::Vertical:
+            ss << L"direction: vertical, ";
+            break;
+        case SplitState::Horizontal:
+            ss << L"direction: horizontal, ";

```

Comment:
`RS_`

Reply:
I was intentionally _not_ localizing these boys, to match the `key: value`s from the json directly. If we think they _should_ be localized, these are easy, but some of the other ones in NewTerminalArgs might be harder.